dns/dns_updater.go

```diff
@@ -144,17 +159,20 @@ func (u *updater) Health() error {
 }
 
 func (u *updater) Update(entries controller.IngressEntries) error {
-	aRecords, err := u.r53.GetARecords()
+	route53Records, err := u.r53.GetRecords()
 	if err != nil {
 		log.Warn("Unable to get A records from Route53. Not updating Route53.", err)
 		failedCount.Inc()
 		return err
 	}
 
-	aRecords = u.determineManagedRecordSets(aRecords)
-	recordsGauge.Set(float64(len(aRecords)))
+	// Flatten Alias (A) and CNAME records into a common structure
+	var records []record = u.flattenRoute53ResourceRecordSet(route53Records)
```

Comment:
`records := u.flattenRoute53ResourceRecordSet(route53Records)` would be the canonical syntax here I think?